charms/jupyter-ui/requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+-r requirements.txt
```

Comment:
Test dependencies are installed in the root directory of the repository where linting, tests and coverage are executed by tox.